torch/linalg/__init__.py

```diff
@@ -821,8 +821,8 @@
     https://www.netlib.org/lapack/lug/node128.html
 """)
 
-ldl_factor_ex = _add_docstr(_linalg.linalg_ldl_factor_ex, r"""
-linalg.ldl_factor_ex(A, *, hermitian=False, check_errors=False, out=None) -> (Tensor, Tensor, Tensor)
+ldl_factor = _add_docstr(_linalg.linalg_ldl_factor, r"""
+linalg.ldl_factor(A, *, hermitian=False, out=None) -> (Tensor, Tensor)
 
 Computes a compact representation of the LDL factorization of a Hermitian or symmetric indefinite matrix.
```

Comment:
```suggestion
Computes a compact representation of the LDL factorization of a Hermitian or symmetric (possibly indefinite) matrix.
```
either that or simply delete the "indefinite", as this function also works when the input matrix is positive/negative definite.